src/components/progress-report/workflow/transitions.ts

```diff
@@ -17,36 +17,71 @@ export const Transitions = defineTransitions({
     to: Status.InProgress,
     label: 'Start',
     type: Type.Approve,
-    notify: 'FPM',
+    notify: {
+      TeamMembersWithRoles: [
+        Role.ProjectManager,
+        Role.RegionalDirector,
+        Role.FieldOperationsDirector,
+      ],
+      Emails: ['MONDAY_EMAIL_DISTRO', 'andre_turner@tsco.org'],
+    },
   },
   'In Progress -> Pending Translation': {
     from: Status.InProgress,
     to: Status.PendingTranslation,
     label: 'Submit for Translation',
     type: Type.Approve,
-    notify: ['Next', 'Translator'],
+    notify: {
+      TeamMembersWithRoles: [
+        Role.ProjectManager,
+        Role.RegionalDirector,
+        Role.FieldOperationsDirector,
+      ],
+      Emails: [''],
```

Comment:
Empty strings should be avoided to declare a lack of a value. nulls or undefined should be used instead where it's a single value. Where it's a list, an empty list works better. In this case the property could just be optional and omitted where not needed.